src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import '../../App.scss';
+
+const notMarried = 'I am not married';
```

Comment:
const notMarried is related only to Person component, so it's better to move it inside the component